wiki/deploy.yaml

```diff
@@ -0,0 +1,57 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: wiki
+  namespace: default
+  labels:
+    app: wiki
+  annotations:
+    fluxcd.io/automated: 'true'
+spec:
+  replicas: 1
+  revisionHistoryLimit: 3
+  selector:
+    matchLabels:
+      app: wiki
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      name: wiki
+      labels:
+        app: wiki
+    spec:
+      containers:
+      - name: wiki
+        image: requarks/wiki:2
+        imagePullPolicy: IfNotPresent
+        env:
+          - name: DB_TYPE
+            value: "postgres"
+          - name: DB_HOST
+            value: "joost.chnet"
+          - name: DB_PORT
+            value: "5432"
+          - name: DB_USER
+            value: "wiki"
+          - name: DB_PASS
+            valueFrom:
+            secretKeyRef:
+              name: wiki
+              key: datasource-password
+          - name: DB_NAME
+            value: "wiki"
+          - name: DB_SSL
```

Comment:
Ik weet niet of dit gaat werken als je niet ergens het SSL CA cert neerzet. Als dat veel gedoe is, zonder SSL laten werken?

Reply:
Weet het ook niet volgens de docu hoef je de SSL CA cert niet te specificeren en probeert die daar zelf uit te komen.

Maar anders kan het denk ik ook wel zonder SSL